indico/modules/categories/__init__.py

```diff
@@ -86,6 +86,8 @@ def _sidemenu_items_old(sender, category, **kwargs):
 def _sidemenu_items(sender, category, **kwargs):
     yield SideMenuItem('content', _('Content'), url_for('categories.manage_content', category),
                        100, icon='eye')
+    yield SideMenuItem('protection', _('Protection'), url_for('categories.manage_protection', category), 70,
+                       icon='shield')
```

Comment:
I'd put the 70 in this line. More readable to have it consistent for all menu items